src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * To style the map
+ */
+
+function createMap() {
+
+	// Create a new StyledMapType object, passing it an array of styles,
+	// and the name to be displayed on the map type control.
+	var styledMapType = new google.maps.StyledMapType(
```

Comment:
```suggestion
	const styledMapType = new google.maps.StyledMapType(
```

Reply:
@potay I searched online and saw that the "difference is that once you’ve assigned a value to a variable using const, you can’t reassign it to a new value" whereas you can with var. May I ask why in this case const is suggested over var?

Reply:
I think the general rule is that use const whenever possible. If there is no intention of modifying the variable, then a const should be used so that when it is somehow modified, an error statement would be resulted instead of continuing with some unintended behaviour.